src/DataViews/include/DataViews/AppInterface.h

```diff
@@ -106,6 +107,9 @@ class AppInterface {
 
   virtual void Disassemble(uint32_t pid, const orbit_client_protos::FunctionInfo& function) = 0;
   virtual void ShowSourceCode(const orbit_client_protos::FunctionInfo& function) = 0;
+
+  [[nodiscard]] virtual const orbit_statistics::BinomialConfidenceIntervalEstimator*
```

Comment:
I prefer getters to return const references instead of pointers to const whenever possible. It's the norm for getters to return const references, so if I see a pointer instead of a reference I have to assume that it could be `nullptr`.